CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Changed
+* **css** `dso-icons.svg`: Inline `<style>` elementen met `color: <kleur>` vervangen met `fill="<kleur>"` tbv. Content-security-policy issue met SVG spritesheet voor iconen ([#1600](https://github.com/dso-toolkit/dso-toolkit/issues/1600))
```

Comment:
Ik denk dat je een rebase foutje hebt gemaakt, dit issue is vorige release opgeleverd.